packages/application/style/scrollbar.css

```diff
@@ -17,13 +17,13 @@
  * will match lower in the tree, and so will override the above */
 [data-jp-theme-scrollbars='true'] .CodeMirror-hscrollbar,
 [data-jp-theme-scrollbars='true'] .CodeMirror-vscrollbar {
-  scrollbar-color: rgba(var(--jp-scrollbar-thumb-color), 0.5) transparent;
+  scrollbar-color: rgb(var(--jp-scrollbar-thumb-color) 0.5) transparent;
```

Comment:
Is this change correct?

The variable is defined in packages/theme-dark-extension/style/variables.css (around L397) as:
```
--jp-scrollbar-thumb-color: 88, 96, 97
```

So this will result in `rgb(88, 96, 97 0.5)` that is incorrect isn't it?

But arguably the variable should contain the full color (but this is gonna break the CSS API)...